src/OpenTK.Platform/Toolkit.cs

```diff
@@ -161,6 +163,11 @@ public static IVulkanComponent Vulkan
             }
         }
 
+        /// <summary>
+        /// Interface for processing platform events.
+        /// </summary>
+        public static IEventComponent Event => Initialized ? _eventComponent! : _eventComponentProxy;
```

Comment:
I think it would be easier to have a static class called `Toolkit.Event` that always exists and has a static event, but I suppose static events open users up to pitfalls when it comes to GC.

We could also consider making the user send their delegate as part of `ToolkitOptions` but I think that is somewhat limiting in that only really one place can subscribe to event, which may be fine, but may also be more work than we want.

So I think static class that static delegate is probably fine as a first revision, and we can make the functions themselves be delegated to some interface that we create so that the behavior can be changed dynamically.